### ISSUE TITLE
Pass command line arguments to deploy task

### ISSUE THREAD

**Issue by omriperl** [wighawag/hardhat-deploy#255]
Hi,
Currently it is possible to define CLI arguments for a hardhat script.
This feature doesn't seem to be supported in hardhat deploy and would be extremely useful.

**Comment by tuler** [comment]
You can still write your own hardhat tasks that calls `hre.run("deploy")`, and define params in your task.

However, you should reconsider to commit configuration to git, because reproducible deploys are important.
The deploy "script" should be idempotent.

**Comment by CJ42** [comment]
I agree with @omriperl . It would be useful to be able to provide constructor arguments via the CLI.
This would be useful especially for running the deploy scripts in the CI.

For instance, `hardhat verify` allows to pass constructor arguments directly in the same command, or via an external file using the `constructor-args` flag.

If both tasks `hardhat deploy` and `hardhat verify` could have this flag, it would help a lot indeed.
